src/endpoints/transactions.rs

```diff
@@ -97,6 +97,12 @@ pub enum DeclineReason {
     /// Incorrect CVC code used
     InvalidCvc,
 
+    /// Strong Customer Authentication blocking 'not present' transaction
+    ScaNotAuthenticatedCardNotPresent,
+
+    /// Requires SCA
+    StrongCustomerAuthenticationRequired,
+
     /// All other errors
     Other,
```

Comment:
Certainly begs the question of why there is an 'Other' variant if it doesn't deserialise an unknown reason to 'Other'...